src/app/main-api-handler.ts

```diff
@@ -251,6 +251,11 @@ ipcMain.on(
       case apiCmds.setIsMana:
         if (typeof arg.isMana === 'boolean') {
           windowHandler.isMana = arg.isMana;
+          // Update App Menu
+          const appMenu = windowHandler.appMenu;
+          if (appMenu && windowHandler.isMana) {
+            appMenu.buildMenu();
+          }
```

Comment:
@KiranNiranjan Do we need to rebuild the menu here since we already do it on `did-finish-load`?

Reply:
Not required unless we add or remove any menu items after the app is loaded

Reply:
@VishwasShashidhar  @KiranNiranjan  We are changing the menu action (assign zoomIn/Out method to menu) when `isMana` is set to true.

Reply:
Sounds good @parthprajapati05. I'll merge the PR once you have unit tests ready.